doc/manual/src/release-notes/rl-next.md

```diff
@@ -40,3 +40,6 @@
 
   As before, the old output will continue to work, but `nix flake check` will
   issue a warning about it.
+
+* Items in the apps output schema must be of type "app" and other uses of `nix
+  run` must be of type derivation.
```

Comment:
```suggestion
* `nix run` is now stricter wrt what it accepts:
    * Members of `apps` are now required to be apps (as defined in [the manual](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-run.html#apps))
    * Member of `packages` or `legacyPackages` must be plain derivations
```

Which also makes me realize that the [description](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-run.html#description) section in the manual would need to be updated

Reply:
huh, but "member[s] (sic.) of `legacyPackages` must be plain derivations" is not really correct, because (nested) attrsets containing derivations are/should also be allowed there, which is afaik the purpose of `legacyPackages` in contrast to `packages`.

Reply:
re-reading the src/nix/run.md and it seems the new behavior is already documented:

    An app is specified by a flake output attribute named `apps.<system>.<name>"...